roles/update_containers/templates/update_containers.j2

```diff
@@ -6,6 +6,7 @@ metadata:
 spec:
   customContainerImages:
 {% if cifmw_update_containers_openstack | bool %}
+    agentImage: {{ cifmw_update_containers_registry }}/{{ cifmw_update_containers_org }}/openstack-baremetal-operator-agent:{{ cifmw_update_containers_tag }}
```

Comment:
We can move this a seperate var just like ansibleee

Reply:
ack